README.md

```diff
@@ -129,3 +129,22 @@ Result:
     }
 ]
 ```
+
+### Testing
+
+You can create test directory and configure dependencies like this (working directory is the root of this repo):
+```
+# create meteor app for testing
+meteor create --release 1.8.1-rc.1 --bare test
```

Comment:
These instruction will probably need some updating.

Reply:
Correct, it ended up in this PR by mistake anyway.
I'll fix it, just notify me if you want it to be removed.

Reply:
If you can update it, that would be better.

Reply:
Updated:
https://github.com/cult-of-coders/grapher/pull/403/commits/8c9e1c4596fe6af57ad2346d641729f747ed0461